src/store/modules/user/actions.ts

```diff
@@ -19,9 +19,33 @@ const actions: ActionTree<UserState, RootState> = {
       const resp = await UserService.login(username, password)
       if (resp.status === 200 && resp.data) {
         if (resp.data.token) {
-            commit(types.USER_TOKEN_CHANGED, { newToken: resp.data.token })
```

Comment:
We should add this to else block of condition to check if permissionId exist. This will handle the case when the permissionId is not there in environment, still the user will be loggedIn.